support/Python/Schedule.py

```diff
@@ -110,9 +111,18 @@ def _copy_submit_script_template(
     return dest
 
 
-# Write `pathlib.Path` objects to YAML as plain strings
-def _path_representer(dumper: yaml.Dumper, path: Path) -> yaml.nodes.ScalarNode:
-    return dumper.represent_scalar("tag:yaml.org,2002:str", str(path))
+# Write following objects to YAML
+#  - `pathlib.Path`
+#  - np.float64
+def _path_representer(dumper: yaml.Dumper, to_dump) -> yaml.nodes.ScalarNode:
+    if isinstance(to_dump, Path):
+        return dumper.represent_scalar("tag:yaml.org,2002:str", str(to_dump))
+    elif isinstance(to_dump, np.float64):
+        return dumper.represent_scalar(
+            "tag:yaml.org,2002:str", str(float(to_dump))
```

Comment:
why string? should probably be represented as a float in yaml, like Python's `float`

Reply:
Because I tried it with just `float` and got an error

Reply:
If you write it as a string it will read back as a string, which is wrong (and could be achieved simply by casting the float to a string before writing). I think the only acceptable behavior is if the `np.float64` reads back in as a `np.float64` or a `float`.